<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlTab</class>
 <widget class="QWidget" name="ControlTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>371</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1" columnminimumwidth="0,0,0,0">
    <item row="3" column="1">
     <widget class="QSpinBox" name="spnSAngle"/>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spnVAngle"/>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="btnExecute">
      <property name="statusTip">
       <string>执行采集</string>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QSpinBox" name="spnConditionValue"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>停止条件</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>初始水平角度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spnHAngle"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>初始垂直角度</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>开始角度</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spnEAngle"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>间隔角度</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spnIAngle"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>间隔时间</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QComboBox" name="cmbCollectMode">
      <item>
       <property name="text">
        <string>fixed_point</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>vertical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>driving</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="cmbStopType">
      <item>
       <property name="text">
        <string>次数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>时间(s)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spnITime"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>结束角度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="5">
     <widget class="QListWidget" name="lwdActionList"/>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="btnClear">
      <property name="statusTip">
       <string>清除所有动作</string>
      </property>
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="btnAdd">
      <property name="statusTip">
       <string>增加一条动作</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>371</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QSpinBox" name="spnChannelNumber">
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>通道号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>累加次数</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>采样深度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>脉冲宽度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spnSamplingDepth">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spnAccumulativeFrequency">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65536</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spnPulseWidth">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="btnConfirm">
      <property name="statusTip">
       <string>设置采集卡参数</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>254</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>垂直</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>水平</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="lblCurrentVerticalAngle">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="btnDown">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="lblCurrentHorizontalAngle">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QPushButton" name="btnUp">
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btnLeft">
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnRight">
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
